### DISCUSSION TITLE
Liability

### DISCUSSION THREAD

**Discussion by ImproveAbility** [ATMakersOrg/EyeAlert#2] [General]
I also want to bring up this message that came across the RESNA listserv when I posted my problem statement.  

From Ray Grott - "When I've been asked in the past to create unique alerting systems for people who are medically fragile, I've declined as soon as it was clear that this was for emergency situations (vs signaling for help with a non-critical need.) That's when the alarm bells go off regarding liability and risk of getting blamed when something goes wrong. I also know that waivers and attestations that a custom alerting system is not for use in emergency situations won't stand in the way of a lawyer."

My thinking is that this is meant to be a backup for the eye gaze system.  And, if I was in the users shoes I would rather have a back up than nothing, but since we are dealing with severely disabled people (most of whom will have no other way to communicate) I think we should think through the liability issues.  

This does matter differently depending on how the device is supplied.  In my case, we would be providing the device for purchase, so that probably opens us up more to liability than if a charity donated an item.  I know this is a bit off topic with regards to the technical issues, so that is why I started a new discussion.  I would love to hear how Bill has handled this with some of the unique solutions that AT makers has been a part of.  

For one of the clients that this device is for, his caregiver is basically deaf.  I did have the thought at one time - maybe I should make some sort of way to alert her that the vent alarm is going off.  But, that is a line that I do not want to cross.  My thought there is that the bad thing is already happening - as the caregiver of someone with a severe disability, we should be doing whatever we can to prevent the bad thing from happening.  So, I do not want to be part of that situation.  The device that we are talking about is meant to be a back up to an eye gaze so that the user can get a caregiver's attention before the bad thing happens, so that seems different to me.

The issue of liability is something that I think about regularly with some of the other custom work that we do (mainly switch adapting items), so I guess I would like to hear others thoughts on the topic.

**Comment by ATMakersBill** [comment]
Well you’re certainly not wrong to be thinking about it. We had a big discussion about it recently that is worth watching. https://youtu.be/Po5wruZw1lc

When I told Chris about this project his first response was “didn’t you just say we weren’t going to do emergency alerts?”

So you’re friend is right that waivers are no longer effective at limiting liability (other than to deter small claims… once a lawyer is involved waivers don’t help). They once were but are not anymore. 

Warnings on the other hand are still effective. The key distinctions are that warnings try to prevent a dangerous situation: waivers just say we’re not responsible for any damages. 

You’re also right in that selling a product has a completely different liability landscape (called strict liability) versus donated items which are covered under negligence laws. The standards are different: strict liability says that if the end user is using your product as intended and it fails, you are presumed to have some liability. Negligence is different: it considers whether your actions when making and deploying the device were reasonable and whether you met a standard of care. 

This is why we don’t sell devices. I would like to find a way around this but haven’t. 

All that said, I think we should proceed. Designing this device and writing the software doesn’t create liability. Even posting instructions and videos doesn’t.  Only deploying it involves negligence concerns and only selling it involves strict liability. 
We can make these, publish the designs (which provides prior to protect against patents), and donate a set of prototypes for testing. Document the results. Hell: do a study and get published in ATOB. 
We add warnings about how Not to use these devices to protect against negligence claims. 

At that point if this is successful and we want to sell these we can work with AT vendors like Jay Beavers at Tolt or Glenn Dobbs at LoganTechvto have them made commercially under their liability coverage.

**Reply by ImproveAbility** [reply]
Thanks Bill - I will watch the video when I can.  This is very helpful info.  I already pay a King's ransom for liability insurance, so not trying to add to that bill!  I am sure we can find a way to offer a solution once we have one.

**Reply by ImproveAbility** [reply]
Hello there - I got a chance to watch the video - great stuff!  I have some specific things that I want to share with Bill - I will send you a separate email.  But, I think it is very much worth watching for this group to get in the right mindset.
